client/src/utils/group.ts

```diff
@@ -0,0 +1,15 @@
+export function groupContingousIndexes(indexes: number[]) {
```

Comment:
might be worth adding the same test cases I added to the python

Reply:
Will do.

Reply:
Added tests 👍